### DISCUSSION TITLE
Avatar

### DISCUSSION THREAD

**Discussion by adolfoalamo** [mozilla/hubs#5432] [Avatar Questions]
How can I add and imported as admin an avatar as feature so other users can use the avatar no only me.  I checked and the imported avatar shows only under MyAvatar tab and is not available as feature.  I am admin of the Hubs Cloud.

**Comment by matthewbcool** [comment]
I'll start with importing an avatar link here:
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/4493657/168686322-17504a7a-1fc6-4e19-9baf-51a56309d15c.png">

<img width="1587" alt="image" src="https://user-images.githubusercontent.com/4493657/168686483-3be0e43c-6118-4274-8316-77e1fb017940.png">
Now I click import

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/4493657/168686551-7cd1ff4d-8dd0-4305-af97-abc745f14505.png">

You should see your avatar in the approved avatar tab.

<img width="1043" alt="image" src="https://user-images.githubusercontent.com/4493657/168687019-ba55278a-95be-40f2-9f8f-d0f7d649246d.png">

Using the above method you can import avatars and [packs of avatars](https://hubs.mozilla.com/docs/hubs-cloud-asset-packs.html ) for everyone to use.
